cmd2/cmd2.py

```diff
@@ -1599,12 +1599,11 @@ def onecmd_plus_hooks(self, line: str, *, add_to_history: bool = True, py_bridge
         stop = False
         try:
             statement = self._input_line_to_statement(line)
-        except EmptyStatement:
+        except (EmptyStatement, ValueError) as ex:
+            if isinstance(ex, ValueError):
```

Comment:
Is there a specific place in the code we expect shlex.split to potentially throw a ValueError exception?  If there are one or two such places, I would recommend catching the exception there and re-raising it as a custom exception called something like **Cmd2ShlexException**.